crates/core/component/stake/src/validator/bonding.rs

```diff
@@ -71,7 +75,9 @@ impl TryFrom<pb::BondingState> for State {
                 } else {
                     anyhow::bail!("unbonding epoch should be set for unbonding state")
                 };
-                Ok(State::Unbonding { unbonding_epoch })
+                Ok(State::Unbonding {
+                    unbonds_at_epoch: unbonding_epoch,
+                })
```

Comment:
i think these expressions could be updated to follow the `unbonds_at_epoch` renaming, keeping the shorthand syntax from before. not a blocker.